receiver/prometheusreceiver/README.md

```diff
@@ -104,6 +104,63 @@ receivers:
               - targets: ['0.0.0.0:8888']
 ```
 
+## Complete Configuration Example
+
+The following example demonstrates a complete end-to-end configuration showing how to use the Prometheus receiver with processors and exporters in a service pipeline:
+
+```yaml
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: 'my-service'
+          scrape_interval: 15s
+          static_configs:
+            - targets: ['localhost:9090']
+          metric_relabel_configs:
+            - source_labels: [__name__]
+              regex: 'http_request_duration_seconds.*'
+              action: keep
+
+processors:
+  batch:
+    timeout: 10s
+    send_batch_size: 1000
+  resource:
+    attributes:
+      - key: service.name
+        value: my-service
+        action: upsert
+      - key: deployment.environment
+        value: production
+        action: upsert
+
+exporters:
+  otlp:
+    endpoint: otel-collector:4317
+    tls:
+      insecure: true
+  prometheusremotewrite:
+    endpoint: https://prometheus:9090/api/v1/write
+    external_labels:
```

Comment:
we should probably just use resource attributes here instead of external_labels?

I also would rather keep the config for this exporter as simple as possible, given it may change in the future (and probably won't be updated here)